includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -122,15 +124,19 @@ public function should_use_sensei_theme() {
 
 		$course_id = \Sensei_Utils::get_current_course();
 
-		if ( null === $course_id ) {
+		if ( empty( $course_id ) ) {
 			return false;
 		}
 
-		if ( ! $this->has_sensei_theme_enabled( $course_id ) ) {
-			return false;
+		if ( self::has_sensei_theme_enabled( $course_id ) ) {
```

Comment:
Maybe adding both conditionals in one here would make it more clear?

```
if ( self::has_sensei_theme_enabled( $course_id ) || Sensei_Course_Theme:: is_sensei_theme_preview_mode_allowed( $course_id ) ) {
```

Reply:
Sure, updated. 341e6e5fffca51b3b66d04c3fe72633e64149d9a